pagopa/io-dev-api-server
# [IOPID-2359] remove remote FF session refresh

> [!Warning] 
> This PR depends on https://github.com/pagopa/io-services-metadata/pull/847

## Short description
Delete remote session refresh FF

## How to test
Run the application using local environment and iand using proxyman or any tool that can intercept the fetch calls made by the app you'll see that in this one here: `http://proxyman.local:3000/static_contents/status/backend.json?ms=*` the value `sessionRefresh` is not entered (as you can see from the screen)

<img width="250" alt="response without sessionRefresh" src="https://github.com/user-attachments/assets/497b3470-232b-4de3-b6ae-e3d95133d1e5">
